howso/remove_cases.amlg

```diff
@@ -2,29 +2,27 @@
 (null
 
 	;removes all cases that match the specified conditions from trainee
-	;
-	;parameters:
-	; case_indices: a list of session id and training index tuples that specify which cases are to be removed
-	; precision: flag, whether to query for 'exact' matches; if set to 'similar' will move num_cases with the most similar values. Ignored if case_indices is specified.
-	; condition: assoc of feature->value(s) (no value = must have feature, one value = must equal exactly the value, two values = inclusive between). Ignored if case_indices is specified.
-	; condition_session: optional, if specified, ignores condition and instead operates on all cases that were trained with this session id. Ignored if case_indices is specified.
-	; num_cases: optional, limit on the number of cases to move; If set to zero there will be no limit. Ignored if case_indices is specified.
-	;		If null, will be set to k if precision is "similar" or no limit if precision is "exact". default is null
-	; distribute_weight_feature: name of feature into which to distribute the removed cases' weights to their neighbors.
 	#remove_cases
 	(declare
 		(assoc
 			;{ref "CaseIndices"}
+			;a list of session id and training index tuples that specify which cases are to be removed
 			case_indices (null)
 			;{ref "Condition"}
+			;assoc of feature->value(s) (no value = must have feature, one value = must equal exactly the value, two values = inclusive between). Ignored if case_indices is specified.
 			condition (assoc)
 			;{type "string"}
+			;optional, if specified, ignores condition and instead operates on all cases that were trained with this session id. Ignored if case_indices is specified.
 			condition_session (null)
 			;{ref "Precision"}
+			;flag, whether to query for 'exact' matches; if set to 'similar' will move num_cases with the most similar values. Ignored if case_indices is specified.
```

Comment:
lets change 'move' to 'remove' on this line, this method was formerly "move_cases"...

Reply:
Corrected in 6d0f612